### ISSUE TITLE
Add blocked/muted users & hidden domains list to Settings page

### ISSUE THREAD

**Issue by bclindner** [tootsuite/mastodon#9650]
### Pitch

The muted users, blocked users, and hidden domains lists are in a somewhat un-intuitive place, in the context menu of the compose bar, here: 
![](https://i.imgur.com/5f5CSMe.png)

I propose these lists also be in some way accessible through the Settings page, to ensure users can more easily find them.

I'm no UX designer, but I think the most straightforward approach would be a menu similar to (or replacing) Filters with submenus that provide an interface within the Settings page itself for handling blocks, mutes, domain hides, and filters.

If writing new interfaces for these menus is too much work, then simple redirects to `/web/mutes`, `/web/blocks`, etc. would also work, similar to how the "muted words" link in the above list merely redirects to the Settings page for filters.

### Motivation

I spent an embarrassing amount of time trying to find the block list on my own a few weeks back before I realized I hadn't checked that context menu, and having helped someone with that earlier today as well, I worry that this could go on to become a major point of frustration for other new users as Mastodon grows.

**Comment by MirceaKitsune** [comment]
I agree. This should really be in the Settings / Edit Profile panels too. Perhaps it can be added directly on the Filters page where it's most intuitive? eg: https://mastodon.social/filters

**Comment by debbiegoldsmith** [comment]
It would be really nice to be able to edit it, too, and preemptively block known awful instances (e.g. the new glindr.org, which is specifically for transphobes banned from birdsite). AFAIK the only way to block a domain now is to wait for a toot from that domain, then block via the menu on that toot.

**Comment by konomikitten** [comment]
I just spent ages looking for this this should be in a dedicated menu item.

**Comment by brendanjones** [comment]
That's nuts, I would have never found it there. I was resigned to the fact that the only way to find out your blocks was to download the csv from https://fosstodon.org/settings/export. Definitely needs fixing.

Don't add the block list to https://fosstodon.org/filters like someone suggested above, that doesn't make sense. It's a list of blocked people, not a filter. (if you want to get semantic then yes, a block is a form of filtering out users, but that's not intuitive)

I hate to copy birdsite but I think their settings grouping 'privacy and safety' is logical. My proposal is to create a new settings menu item, and group there filters, blocks and mutes. Like so:

- fosstodon.org/settings/privacy_and_safety 
- - fosstodon.org/settings/privacy_and_safety/filters
- - fosstodon.org/settings/privacy_and_safety/mutes
- - fosstodon.org/settings/privacy_and_safety/blocks
- - fosstodon.org/settings/privacy_and_safety/domain_blocks
[obviously replace `fosstodon.org`, that's just my server]

You could also add https://fosstodon.org/follow_requests here under fosstodon.org/privacy_and_safety , but they would maybe be more logically placed under https://fosstodon.org/relationships

**Comment by brendanjones** [comment]
> If writing new interfaces for these menus is too much work, then simple redirects to /web/mutes, /web/blocks, etc. would also work, similar to how the "muted words" link in the above list merely redirects to the Settings page for filters.

Please don't do this. Honestly that entire ellipsis menu (the three dots) should be removed - almost everything in there can be accessed through another, more logical interaction. The couple of things that can't be accessed elsewhere should have those interactions made. 

The only things that need fixing before that ellipsis menu can be removed:

1. Follow requests `https://fosstodon.org/follow_requests`could go in two places, either in settings below relationships like `https://fosstodon.org/settings/relationships/follow_requests` or on your profile `https://fosstodon.org/@Brendanjones/follow_requests` (to match the current follows/followers links like https://fosstodon.org/@Brendanjones/followers)
2. https://fosstodon.org/pinned should go to `https://fosstodon.org/@Brendanjones/pinned`

Yes, this should go in a new issue, I'll make it and edit this comment when I've made it.

**Comment by grahamperrin** [comment]
[Dealing with unwanted content - Mastodon documentation](https://docs.joinmastodon.org/user/moderating/#mute) was ~~frustrating~~ entertaining: 

> The user has no way of knowing they have been muted.

For too long, I felt that, additionally: 

> The muter has no way of knowing who they muted.

Whilst I'm not sure about #23355 (removal of a menu), I _am_ **certain** that a preferences/settings page must include an area for things such as mutes. 

Consider: 

![image](https://github.com/mastodon/mastodon/assets/192271/ad9bae2e-a993-449b-89f2-22da45a172de)

1. a hamburger menu icon (top left) that does not reveal a menu
2. four **visible** icons for preferences, sanely positioned
3. the fifth route to preferences, **hidden** beneath the ellipsis
4. the preferences above and below the fifth route that are **not within** the preferences in the fifth route
